setup.py

```diff
@@ -52,7 +52,7 @@ def package_files(directory):
     package_data={'': extra_files},
     install_requires=[
         'ruamel.yaml==0.16.10',
-        'markupsafe==2.0.1',
+        'markupsafe==2.1.1',
```

Comment:
This will only be consistent until one of the four things that depends on this in the manager update their deps.